### PR TITLE
HOTFIX: Feat/Show warning on congestion PE-694

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

File: lib/misc/misc.dart [hunk 1]
```diff
@@ -10,3 +10,5 @@ const kTrimTrailingRegex = r'[^\s \.]+$';
     Change back to previous regex with lookbehind when
     Safari supports it.
   */
+
+const mempoolWarningSizeLimit = 1000;
```
Comment (arielmelendez) [inline lib/misc/misc.dart:14]: nit: no trailing newline

File: lib/pages/congestion_warning_wrapper.dart [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import 'package:ardrive/services/services.dart';
+import 'package:ardrive/theme/theme.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+Future<void> showCongestionWarning(
+    BuildContext context, Function() showAppDialog) async {
+  final warnAboutCongestion =
+      await context.read<ArweaveService>().getMempoolsize() >
```
Comment (arielmelendez) [inline lib/pages/congestion_warning_wrapper.dart:12]: How does this perform under poor networking conditions?

**Review by arielmelendez — APPROVED** [review]

File: lib/pages/congestion_warning_wrapper.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/congestion_warning_wrapper.dart:12]: It would lag. Should I add a timeout?

**Review by fedellen — APPROVED** [review]

File: lib/pages/congestion_warning_wrapper.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/congestion_warning_wrapper.dart:13]: UI would freeze because this is all taking place on the same thread.

**Review by arielmelendez — DISMISSED** [review]

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**